src/axom/inlet/docs/sphinx/quick_start.rst

```diff
@@ -0,0 +1,124 @@
+
+Define the schema by using the following functions, on either the main Inlet class, for
+global Tables and Fields, or on individual Table classes, for Tables and Fields under that Table:
+
+========================= ===================
+Name                      Description
+========================= ===================
+addTable                  Adds a Table to the input deck schema with the given name.
+addBool                   Adds a boolean Field to the global or parent Table with the given name.
+addDouble                 Adds a double Field to the global or parent Table with the given name.
+addInt                    Adds a integer Field to the global or parent Table with the given name.
+addString                 Adds a string Field to the global or parent Table with the given name.
+========================= ===================
+
+All possible Tables and Fields that are can be found in the input deck must be defined
+at this step.  The value of the Field is read and stored into the Sidre datastore when you call the appropriate
+add function. Use the ``required`` class member function on the Table and Field class to indicate that
+they have to present in the given input deck. You can also set a default value to each field via the type-safe
+``Field::defaultValue()`` member functions. Doing so will populate the corresponding Fields value
+if the specific Field is not present in the input deck. The following example shows these concepts:
+
+.. literalinclude:: ../../examples/verification.cpp
+   :start-after: _inlet_workflow_defining_schema_start
+   :end-before: _inlet_workflow_defining_schema_end
+   :language: C++
+
+
+.. _inlet_verification_label:
+
+Verification
+------------
+
+This step helps ensure that the given input deck follows the rules expected by the code.  These
+rules are not verified until you call ``Inlet::verify()``.  Doing so will return true/false and
+output SLIC warnings to indicate which Field or Table violated which rule.
+
+As shown above, both Tables and Fields can be marked as ``required``. Fields have two additional
+basic rules that can be enforced with the following ``Field`` class member functions:
+
+========================= ===================
+Name                      Description
+========================= ===================
+validValues               Indicates the Field can only be set to one of the given values.
+range                     Indicates the Field can only be set to inclusively between two values.
+========================= ===================
+
+Inlet also provides functionality to write your own custom rules via callable lambda verifiers.
+Fields and Tables can both register one lambda each via their ``registerVerifier()`` member functions.
+The following example adds a custom verifier that simply verifies that the given ``dimensions`` field
+match the length the given vector:
+
+.. literalinclude:: ../../examples/verification.cpp
```

Comment:
```cpp
  // We expect verification to be unsuccessful since the only Field
  // in vector is x but 2 dimensions are expected
  std::cout << "This should fail due to a missing dimension:\n";
  myInlet->verify() ? std::cout << "Verification was successful\n" 
                    : std::cout << "Verification was unsuccessful\n";
```

Should these use SLIC macros? Or are they intentionally using `cout`?